x-pack/plugins/apm/public/components/alerting/register_apm_alerts.ts

```diff
@@ -172,7 +172,8 @@ export function registerApmAlerts(
     description: i18n.translate(
       'xpack.apm.alertTypes.transactionDurationAnomaly.description',
       {
-        defaultMessage: 'Alert when the latency of a service is abnormal.',
+        defaultMessage:
```

Comment:
`xpack.apm.alertTypes.transactionDurationAnomaly.description` could also be renamed